idaes/models_extra/power_generation/costing/power_plant_capcost.py

```diff
@@ -79,6 +79,12 @@
     between 90% and 97.7% capture rate. It is only valid for a flue gas CO2
     concentration of 4%. The additional cost component of the CO2 system is
     in account 5.1.b and remains unchanged.
+
+    - 6.1 can be used to cost the MEA solvent-based carbon capture system, 
+    using the reference: Kangkang Li, Ashleigh Cousins, Hai You, Paul Feron,
+    Weilang Luo, Jian Chen (2016). Systematic study of aqueous monoethanolamine
+    (MEA)-based CO2 capture process: Techno-economic assessment of the MEA
+    process and its improvements. Applied Energy, 165, 648-659.
  """
 
 custom_costing_exponents = {
```

Comment:
This file is getting too large, can we move the custom_costing_exponents to a new file and import these two dictionaries into the get_PP_costing method?

Reply:
I'll move them into a new file